scheduler/replica_scheduler.go

```diff
@@ -291,6 +302,11 @@ func (rcs *ReplicaScheduler) filterNodeDisksForReplica(node *longhorn.Node, disk
 	multiError = util.NewMultiError()
 	preferredDisks = map[string]*Disk{}
 
+	allowEmptyDiskSelectorVolume, err := rcs.ds.GetSettingAsBool(types.SettingNameAllowEmptyDiskSelectorVolume)
+	if err != nil {
+		logrus.Errorf("Failed to get allow empty tag volume setting: %v", err)
```

Comment:
return error here and no need to log error here, because the caller will handle the returned error.